sdk/storage/Azure.Storage.Blobs/tests/PageBlobClientTests.cs

```diff
@@ -1505,7 +1505,7 @@ public async Task GetPageRangesDiffAsyncIfTags_Failed()
         }
 
         [Test]
-        [PlaybackOnly("Not possible to programmatically create a managed disk storage account")]
+        [Ignore("Not possible to programmatically create a managed disk storage account")]
```

Comment:
what's the plan to bring back this test?

Reply:
This could be an issue.  We can't make this a regular unit test because it is not possible to programmatically create a managed disk storage account.  We would need to get a managed disk storage account to re-record, and I'm not sure how we would do this.